src/Event/HotCodeReloaderWorkerStartListener.php

```diff
@@ -48,4 +49,12 @@ public function __invoke(WorkerStartEvent $event): void
             $server->reload();
         });
     }
+
+    /**
+     * @internal For unit testing static dependency only.
+     */
+    protected function tick(int $ms, callable $callback): void
+    {
+        Timer::tick($ms, $callback);
```

Comment:
I'm wondering if we should alter this based on detected features; i.e., only use this if it exists, otherwise fallb ack to the `tick()` method on the server instance.

We can't guarantee the Swoole version the user is on...

Reply:
Timer existed in version 4. As far as I was able to determine Server::tick() was introduced as a convenience alias together with Timer and was dropped in v5 along with several other aliases for Timer

Reply:
I can definitely say that `Timer::tick()` existed in swoole 4.3.0 in 2019. Before that I don't know where to look in the code

Reply:
That should be fine, then. Thanks, @Xerkus !